src/resources/elements/primeDesignSystem/pform-input/pform-input.scss

```diff
@@ -34,6 +34,7 @@ pform-input {
     white-space: pre-line;
     line-height: 24px;
     color: $Neutral03;
+    margin-bottom: 10px;
```

Comment:
this makes me nervous that it can affect a lot of places.  Have you checked them all?

Reply:
> fix is save, because only TokenDetails uses label-description

Yup.
I did a search for `label-description`

Reply:
To be safe, since only token details uses label-description, lets move this into there.  I have been dealing with issues with pform-input (when the label and input are horizontally oriented) and really prefer not to mess with this component.  Is too fragile ATM.

Reply:
lcoalized to token-details


[styles(tokenDetails): add bottom margin to label description](https://github.com/PrimeDAO/prime-deals-dapp/pull/620/commits/ee7464145066f62e5b3925040d0c9cbaece9e373)